docs/data-companion/quick-start.md

```diff
@@ -0,0 +1,269 @@
+
+The first step is to set the address for the privileged services, for example:
+```
+#
+# Configuration for privileged gRPC endpoints, which should **never** be exposed
+# to the public internet.
+#
+[grpc.privileged]
+# The host/port on which to expose privileged gRPC endpoints.
+laddr = "tcp://0.0.0.0:26091"
+```
+
+In the `[grpc.privileged.pruning_service]` section, ensure the value `enabled` is set to `true`
+
+```
+#
+# Configuration specifically for the gRPC pruning service, which is considered a
+# privileged service.
+#
+[grpc.privileged.pruning_service]
+
+# Only controls whether the pruning service is accessible via the gRPC API - not
+# whether a previously set pruning service retain height is honored by the
+# node. See the [storage.pruning] section for control over pruning.
+#
+# Disabled by default.
+enabled = true
+```
+
+## Fetching data
+
+CometBFT has recently introduced new gRPC services to facilitate data retrieval via the gRPC endpoint services.
+These new services, namely, `version, `block` and `block results`, offer a more efficient and faster means of fetching data
+as compared to the RPC endpoint. While the RPC endpoint is still a viable option, gRPC will be the preferred choice going forward.
+
+### Fetching **Block** data
+
+In order to retrieve `block` data using the gRPC block service, you have to ensure the gRPC endpoint and the service is
+enabled as described in the section above.
+
+Once the service has been enabled, the Golang gRPC client provided by CometBFT can be utilized to retrieve data from the node,
+such as retrieved a block by its height.
+
+This client code is a convenient option for retrieving data, as it allows for requests to
+be sent and responses to be managed in a more idiomatic manner. However, if necessary, the proto version can also be used directly.
+
+Here is an example code to retrieve a block by height:
+```
+import (
+     "github.com/cometbft/cometbft/rpc/grpc/client"
+)
+
+ctx := context.Background()
+
+// Service Client
+addr := "0.0.0.0:26090"
+conn, err := client.New(ctx, addr, client.WithInsecure())
+if err != nil {
+    // Do something with the error
+}
+
+block, err := conn.GetBlockByHeight(ctx, height)
+if err != nil {
+    // Do something with the error
+} else {
+    // Do something with the `block`
+}
+
+```
+
+### Fetching **Block Results** data
+
+To fetch `block results` you can use a similar code as the previous one but just invoking the method to that retrieves
+block results
+
+Here's an example:
+```
+blockResults, err := conn.GetBlockResults(ctx, height)
+if err != nil {
+    // Do something with the error
+} else {
+    // Do something with the `blockResults`
+}
+
+```
+
+### Latest height stream
+
+There is a new way to subscribe to a stream of new blocks with the Block service. Previously, you could connect and
+subscribe to new block events using websockets through the RPC service.
```

Comment:
A benefit of this is that now you can subscribe only for the latest height so your gRPC endpoint is not transferring entire blocks to keep you up to date.. You can then fetch the blocks at your own pace via the `getBlockByHeight` endpoint .

Reply:
added